docs/config_entries_config_flow_handler.md

```diff
@@ -113,6 +113,10 @@ A Unique ID is used to match a config entry to the underlying device or api. The
 
 - IP Address
 - Device Name
+- Hostname
+- URL
+- Email addresses
+- Usernames
```

Comment:
I suggest we remove these from this list. We mention them above that they are sometimes acceptable. That's enough. Having them here to is confusing.
```suggestion
```

Reply:
Good call